packages/framework-info/src/context.js

```diff
@@ -1,30 +1,41 @@
 import { cwd, version as nodejsVersion } from 'process'
 
-import isPlainObj from 'is-plain-obj'
 import { locatePath } from 'locate-path'
 import { readPackageUp } from 'read-pkg-up'
 
-export const getPackageJson = async (projectDir) => {
+/*
+interface PackageJsonInfo {
+  packageJson?: PackageJson
+  packageJsonPath?: string
+}
+
+interface Context extends PackageJsonInfo {
+  pathExists: (path: string) => Promise<boolean>
+  nodeVersion: string
+}
+*/
+
+export const getPackageJson = async (projectDir /*: string*/) /*: Promise<PackageJsonInfo>*/ => {
   try {
     const result = await readPackageUp({ cwd: projectDir, normalize: false })
     if (result === undefined) {
       return {}
     }
 
-    const { version, packageJson, path: packageJsonPath } = result
+    const { packageJson, path: packageJsonPath } = result
 
-    if (!isPlainObj(packageJson)) {
-      return { packageJsonPath }
-    }
-
-    return { version, packageJson, packageJsonPath }
```

Comment:
`version` does not exist in the result and packageJson is always an object
https://github.com/sindresorhus/read-pkg-up/blob/main/index.js

Reply:
proper types would have caught that

Reply:
yes, i tired to convert this file to ts, but this does not work yet as stated above.